packaging/hudi-timeline-server-bundle/pom.xml

```diff
@@ -170,12 +170,96 @@
                             <exclude>META-INF/*.SF</exclude>
                             <exclude>META-INF/*.DSA</exclude>
                             <exclude>META-INF/*.RSA</exclude>
-                            <!-- Use this jar's NOTICE and license file -->
-                            <exclude>META-INF/NOTICE*</exclude>
-                            <exclude>META-INF/LICENSE*</exclude>
+                            <exclude>META-INF/services/javax.*</exclude>
                         </excludes>
                     </filter>
                 </filters>
+                <artifactSet>
+                  <includes>
+                      <include>org.apache.hudi:hudi-common</include>
+                      <include>org.apache.httpcomponents:httpclient</include>
+                      <include>org.apache.httpcomponents:httpcore</include>
+                      <include>org.mortbay.jetty:jetty</include>
+                      <include>org.mortbay.jetty:jetty-util</include>
+                      <include>com.esotericsoftware:kryo-shaded</include>
+                      <include>com.esotericsoftware:minlog</include>
+                      <include>org.objenesis:objenesis</include>
+                      <include>org.apache.hudi:hudi-timeline-service</include>
+                      <include>log4j:log4j</include>
+                      <include>com.fasterxml.jackson.core:jackson-annotations</include>
+                      <include>com.fasterxml.jackson.core:jackson-core</include>
+                      <include>com.fasterxml.jackson.core:jackson-databind</include>
+                      <include>org.apache.httpcomponents:fluent-hc</include>
+                      <include>commons-logging:commons-logging</include>
+                      <include>io.javalin:javalin</include>
+                      <include>org.jetbrains.kotlin:kotlin-stdlib-jdk8</include>
+                      <include>org.jetbrains.kotlin:kotlin-stdlib</include>
+                      <include>org.jetbrains.kotlin:kotlin-stdlib-common</include>
+                      <include>org.jetbrains:annotations</include>
+                      <include>org.jetbrains.kotlin:kotlin-stdlib-jdk7</include>
+                      <include>org.slf4j:slf4j-api</include>
+                      <include>org.eclipse.jetty:jetty-server</include>
+                      <include>javax.servlet:javax.servlet-api</include>
```

Comment:
@vinothchandar : Made one pass of it and cleaned up. For the servlet jar, I have modified the run_server script to fetch this platform jar first time and add to class path to avoid runtime issues. With this, I am able to start timeline server in standalone mode and make request to it